stellarobservatory/quorum_test.py

```diff
@@ -49,7 +49,7 @@ def test_is_quorum():
     """Test is_quorum()"""
     quorum_slices_by_public_key = {
         'A': [{'A', 'B'}, {'A', 'C'}, {'A', 'B', 'C'}],
-        'B': [{'A', 'B'}],
+        'B': [{'A', 'B'}, {'A', 'D'}],
```

Comment:
Hmm, not sure why I've changed this. Probably, can be reverted:
```suggestion
        'B': [{'A', 'B'}],
```